### PR TITLE
Update russian.xml 7.7.2

### PR DESCRIPTION
**Pull request by GOLEM777**
Updated to v7.7.2:
- added new lines of Summary
- added lines Up and Down
- added others lines
NeedToRestartToLoadPlugins
ExitToUpdatePlugins

### CODE REVIEW

File: PowerEditor/installer/nativeLang/russian.xml [hunk 1]
```diff
@@ -319,6 +318,9 @@ Updated to v7.6.6:
 					<Item id="50000" name="Завершение функций"/>
 					<Item id="50001" name="Завершение слов"/>
 					<Item id="50002" name="Подсказка по функциям"/>
+					<Item id="50003" name="Перекл. на предыдущий док."/>
+					<Item id="50004" name="Перекл. на следующий док."/>
+					<Item id="50005" name="Вкл./Выкл. запись Макро"/>
```
Comment (rddim) [inline PowerEditor/installer/nativeLang/russian.xml]: @GOLEM777 

Where I can find these three items above? In which menu?

File: PowerEditor/installer/nativeLang/russian.xml [hunk 2]
```diff
@@ -1032,6 +1034,8 @@ Updated to v7.6.6:
 			<OpenInAdminModeWithoutCloseCurrent  title="Ошибка сохранения"                          message="Файл не может быть сохранен и может быть защищён.&#x0A;Хотите запустить Notepad++ в режиме администратора?"/>
 			<OpenInAdminModeFailed               title="Не удалось открыть в режиме Администратора" message="Notepad++ не может быть открыто в режиме Администратора."/>
 			<ViewInBrowser                       title="Открыть текущий файл в Браузере"            message="Приложение не найдено в вашей системе."/>
+			<ExitToUpdatePlugins                 title="Завершение работы Notepad++"                message="Notepad++ завершит работу для продолжения операции.&#x0A;Программа будет перезапущена после завершения всех операций.&#x0A;Продолжить?"/>
+			<NeedToRestartToLoadPlugins          title="Необходимо перезапустить Notepad++"         message="Необходимо перезапустить Notepad++ для установки загруженных плагинов."/>
```
Comment (rddim) [inline PowerEditor/installer/nativeLang/russian.xml]: Could you add it to the `english.xml` too please

**Comment by donho** [comment]
Thank you for the update.
Sorry, I didn't check if your PR contain all the new added entries. Could you check the file in the following issue :
#5962
and confirm me your PR contain all the new added entries, please?
If not, please add another commit to include all of them.
Thank you.

**Comment by donho** [comment]
@GOLEM777 Could you check the Russian translation about :
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/4317
And do the correction necessary please?

**Comment by GOLEM777** [comment]
Ok, after work, I'll do it.  Сheck аnd fix everything if need.
#4317  - Long ago fixed! )))

**Comment by GOLEM777** [comment]
All is ready!

#5794

All the same, you could not make a small correction for the next release. 
This is an old bug that pursues all versions of the program.
id 50005 - Toggle macro record
id 50004 - Switch to next document
id 50003 - Switch to previous document

**Pull request closed without merging**